cpp/ql/src/external/DefectFilter.qll

```diff
@@ -5,8 +5,8 @@ import cpp
 /**
  * Holds if `id` in the opaque identifier of a result reported by query `queryPath`,
  * such that `message` is the associated message and the location of the result spans
- * column `startcolumn` of line `startline` to column `endcolumn` of line `endline`
- * in file `filepath`.
+ * column `startcol` of line `startline` to column `endcol` of line `endline`
+ * in file `file`.
```

Comment:
As earlier, rename the arguments.

Reply:
Do you know, if that will cause any problems when this is an external predicate?

Reply:
That's a good question. I honestly don't know. Feel free to ignore this review comment.